docs/cc_to_ako.md

```diff
@@ -0,0 +1,89 @@
+# Cloud connector to AKO migration
+## Overview
+Cloud connector(CC) is not supported from Avi Controller versrion 18.2.10 onwards. To migrate existing workload present in kubernetes cluster from cloud connector to AKO, following two features will be used.
```

Comment:
We would have to support cc with all 18.2.x version, including versions > 18.2.10, but not 20.x

Reply:
Addressed. Thanks.